python/dalex/dalex/_explainer/object.py

```diff
@@ -755,10 +758,88 @@ def model_surrogate(self,
 
         return surrogate_model
 
+    def model_fairness(self, protected, privileged, cutoff=0.5, **kwargs):
+        """Creates a dataset level fairness explanation that enables bias detection
+
+        This method returns a GroupFairnessClassification object that for now
+        supports only classification models. GroupFairnessClassification object
+        works as a wrapper of the protected attribute and the Explainer from which
+        `y` and `y_hat` attributes were extracted. Along with an information about
+        privileged subgroup (value in the `protected` parameter), those 3 vectors
+        create triplet (y, y_hat, protected) which is a base for all further
+        fairness calculations and visualizations.
+
+        Parameters
+        -----------
+        protected : np.ndarray (1d)
+            Vector, preferably 1-dimensional nd.array containing strings,
+            which denotes the membership to a subgroup. It doesn't have to be binary.
+            It doesn't need to be in data. It is sometimes suggested not to use
+            sensitive attributes in modelling, but still check model bias for them.
+            NOTE: List and pd.Series are also supported, however if provided
+            they will be transformed into a (1d) np.ndarray with dtype 'U'.
+        privileged : str
+            Subgroup that is suspected to have the most privilege.
+            It needs to be a string present in `protected`.
+        cutoff : float or dict, optional
+            Threshold for probabilistic output of a classifier.
+            It might be:
+                float - same for all subgroups from `protected`
+                dict - individually adjusted for each subgroup
+                       (must have values from `protected` as keys).
+        kwargs :
```

Comment:
If `verbose` is `True` by default it must be a named parameter.

Reply:
Changed default to `False` and added `verbose_cat`  to checks. Now hen user sets `verbose = True` he/she will see information about changing parameters. Hope this is okay